test/licensemaster/enterprise.lic

```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<license>
+  <signature>kGsVWVk5lbo7V6hqKzzXSKmw0KegBMVD3d5J41ddH6Jv+jIEZlqXaz9b5A51HapnaBgi5N/OzWNUxMBVwkBqadeWjgOk385Z1Sn6M5HPq5NYcjrhYAn/zDIILtGw0vHusVLprA0qBsz7Jz9DaCcPgidIOZGC/f/F+k3p/f6IrqMU9X9fDi1xvdq25wIQsiaHZ+vUGEFTmPzu+m8T+pQPICOG8Kw0eYr3MAolpkTKGS6I7Sf8EAMe9j9h6uLI09SEyJdhhH+pmuxEXyaJ8gX58KRzCUVKu6SSTwg1uns43BTVQ85FD7j57c3aSQ05QkvraCYhSdM7YS7mV94r0XWCJw==</signature>

```

Comment:
Do we really want this license in the code base?  I guess its expired now but still...

Reply:
Good catch Jeff. Removing it in the latest commit.